### ISSUE TITLE
Resize at closest border/corner

### ISSUE THREAD

**Issue by ttencate** [ttencate/taekwindow#14]
Converted from [SourceForge issue 1632182](http://sourceforge.net/support/tracker.php?aid=1632182), submitted by ttencate

Resizing is currently always done from the bottom right corner. It would be better to use the closest corner for this, or the closest edge. Needs some more thinking.

**Comment by ttencate** [comment]
Submitted by ttencate

Logged In: YES 
user_id=117084
Originator: NO

I just discovered your project -- nice work! I've been wanting this feature for Windows for ages.

Anyways, this item would be my top feature request. Every X window manager I've used operates like this. Most of them use the middle mouse button as well. For me this is relevant because I use StrokeIt for mouse gestures, which is bound to the RMB.

Again, awesome work, looking forward to 0.2!

**Comment by ttencate** [comment]
Submitted by ttencate

Logged In: YES 
user_id=1679966
Originator: YES

Thanks for the comment! You're actually not the first to request this: an IRL friend of mine has been bugging me about it for some time too. Out of curiosity, how did you discover this project? I've not exactly been promoting it yet.

I'm currently quite busy, but this feature should be easy to implement. Perhaps I'll release a 0.1.1 first with just this feature added. In the meantime I'm looking into an alternative implementation which should be (if it works) a lot faster and more elegant because it lets Windows do all the hard work :)

**Comment by ttencate** [comment]
Submitted by ttencate

Logged In: YES 
user_id=1679966
Originator: YES

I've just uploaded 0.1.1 which resizes at the closest border/corner. Please give it a try. No middle button is used yet; I want to have the configuration tool in place for that first.

**Comment by ttencate** [comment]
Submitted by ttencate

Logged In: YES 
user_id=117084
Originator: NO

Hi -- I was thinking about writing something like this, and after a fruitless Google search I came to sf.net to try and find something like it (wanted to make sure I wouldn't be doing work for nothing). I was sure I wasn't the only one looking for this sort of functionality.

Just downloaded 0.1.1, love it! For now I've remapped StrokeIt to MMB since RMB resize is unusable with it on its default settings.
